### ISSUE TITLE
[Windows] RequestError: getaddrinfo ENOTFOUND registry.npmjs.org

### ISSUE THREAD

**Issue by JMEssex** [tjunnone/npm-check-updates#503]
- [x] I have read the list of [known issues](https://github.com/tjunnone/npm-check-updates#known-issues) before filing this issue.
- [x] I have searched for [similiar issues](https://github.com/tjunnone/npm-check-updates/issues) before filing this issue.

* node version: v11.11.0
* npm version: v6.9.0
* npm-check-updates version: v3.0.5 & v3.0.6
* OS Environment: Windows 7 Enterprise - Service Pack 1 (This is a work computer... **_sigh_**)
* terminal:  [Git BASH for Windows](https://gitforwindows.org/)

---------------------------
After running bash command: `ncu -g`

_This is not reproducible for me on my Macbook Pro Terminal._

I get the following response: Response uses (`npm -g -l silly`) for debugging purposes.
````
$ ncu -g -l silly
Initializing...
Getting installed packages...
globalPackages
{ '@angular/cli': '7.3.5',
  npm: '6.9.0',
  'npm-check-updates': '3.0.6',
  serve: '10.1.2',
  typescript: '3.3.3333' }

Fetching latest versions...
[=========-----------] 45%
(node:9500) UnhandledPromiseRejectionWarning: RequestError: getaddrinfo ENOTFOUND registry.npmjs.org registry.npmjs.org:443
    at ClientRequest.request.once.error (C:\Users\%USERNAME%>\AppData\Roaming\npm\node_modules\npm-check-updates\node_modules\got\source\request-as-event-emitter.js:178:14)
    at Object.onceWrapper (events.js:285:13)
    at ClientRequest.emit (events.js:202:15)
    at ClientRequest.origin.emit.args (C:\Users\%USERNAME%\AppData\Roaming\npm\node_modules\npm-check-updates\node_modules\@szmarczak\http-timer\source\index.js:37:11)
    at TLSSocket.socketErrorListener (_http_client.js:397:9)
    at TLSSocket.emit (events.js:197:13)
    at emitErrorNT (internal/streams/destroy.js:82:8)
    at emitErrorAndCloseNT (internal/streams/destroy.js:50:3)
    at processTicksAndRejections (internal/process/next_tick.js:76:17)
(node:9500) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:9500) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
````

I am sure this has something to do with the `npm-check-updates@3.0.5` error handling release which was 21 hours ago.

Best,
James

**Comment by raineorshine** [comment]
Can you try now?

**Comment by JMEssex** [comment]
Bug fixed! :+1:  As of `npm-check-updates@3.0.8` the command `npm -g` runs as intended.

Thanks for the quick turnaround :)
